components/wpt-results.html

```diff
@@ -219,7 +219,8 @@
       }
 
       _computePathIsATestFile (path) {
-        return path.endsWith('.html') || path.endsWith('.htm')
+        const filePath = /\w+\.\w+/
```

Comment:
This won't work, unfortunately there are a few test directories with dots in them:

```
% find . -type d -name "*\.*"
.
./html/dom/documents/dom-tree-accessors/document.getElementsByName
./webgl/conformance-1.0.3
./webgl/conformance-1.0.3/conformance/ogles/GL2Tests/glGetShaderInfoLog_2.0
./webgl/conformance-1.0.3/conformance/ogles/GL2Tests/glGetProgramiv_2.0
./webgl/conformance-1.0.3/conformance/ogles/GL2Tests/glGetProgramInfoLog_2.0
./tools/pytest/extra/setup-py.test
./tools/pytest/_pytest/vendored_packages/pluggy-0.3.1.dist-info
./tools/pytest/.github
./tools/wptrunner/wptrunner.egg-info
./tools/.hypothesis
./css/CSS2/csswg-issues/submitted/css2.1
./css/css-fonts-3/support/fonts/gsubtest-lookup3.ufo
```

Reply:
A whitelist seems like the best solution for now until we can cross-reference the manifest in the browser (part of #159). We can add the ones @gsnedders mentioned:

```
gsnedders> looking at sourcefile.py, we need at least html, htm, xhtml, xht, xml, svg, py
```